Bottom: 0282831db3967b5fd1c1c8c3b6bf752e4f3bbd79
Top:    4f457f176c3b02306c8021002e41ac8f1e9cdb3c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-10 23:02:01 +0100

Make --enable-dbus default for the GTK and Lesstif HIDs


---

diff --git a/configure.ac b/configure.ac
index 4e9476d..a220a82 100644
--- a/configure.ac
+++ b/configure.ac
@@ -353,7 +353,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
